docs/replication-manager/replication-manager-for-pxc.md

```diff
@@ -0,0 +1,280 @@
+    INSERT INTO `link` VALUES ('DC1','DC3');
+    INSERT INTO `link` VALUES ('DC2','DC1');
+    INSERT INTO `link` VALUES ('DC3','DC1');
+    ```
+
+    Add the weight:
+
+    ```bash
+    INSERT INTO `weight` VALUES('DC1','DC1-1',10); 
+    INSERT INTO `weight` VALUES('DC1','DC1-2',11); 
+    INSERT INTO `weight` VALUES('DC2','DC2-1',9);
+    INSERT INTO `weight` VALUES('DC2','DC2-2',12);
+    INSERT INTO `weight` VALUES('DC3','DC3-1',11);
+    ```
+
+    The node in the cluster with the highest value is the first to be selected.
+
+9. Provision the remote clusters and start replication. 
+
+    On one of the DC1 nodes, for example DC1-1, perform a `mysqldump` with:
+
+    ```bash
+    [root@DC1-1 ~]# mysqldump -u root -p --master-data=2 --single-transaction -R -A -E > dump.sql
+    ```  
+
+10. Compress the file if it's too large. You can use Percona Xtrabackup to compress the data. Copy the backup file to one node in each remote cluster, for example to DC2-1 and DC3-1. 
+
+    To restore the dump run:
+
+    ```bash
+    [root@DC2-1 ~]# mysql -u root -p < dump.sql
+    ```
+
+    and
+
+    ```bash
+    [root@DC3-1 ~]# mysql -u root -p < dump.sql
+    ```
+
+11. Configuring the replication. Set up the first replication links manually. 
+
+    For example, on DC2-1 run:
+
+    ```bash
+    mysql> change master to master_host='WAN IP of DC1-1', master_user='repl', master_password='replpass', MASTER_AUTO_POSITION = 1 for channel 'DC2-DC1';
+    mysql> start slave for channel 'DC2-DC1';
+    ```
+    
+    on DC3-1 run:
+
+    ```bash
+    mysql> change master to master_host='WAN IP of DC1-1', master_user='repl', master_password='replpass', MASTER_AUTO_POSITION = 1 for channel 'DC3-DC1';
+    mysql> start slave for channel 'DC3-DC1';
+    ```
+
+    For the other direction, use DC1-1 for both:
+
+    ```bash
+    mysql> change master to master_host='WAN IP of DC2-1', master_user='repl', master_password='replpass', MASTER_AUTO_POSITION = 1 for channel 'DC1-DC2';
+    mysql> start slave for channel 'DC1-DC2';
+    mysql> change master to master_host='WAN IP of DC3-1', master_user='repl', master_password='replpass', MASTER_AUTO_POSITION = 1 for channel 'DC1-DC3';
+    mysql> start slave for channel 'DC1-DC3';
+    ```
+
+    Now, all the clusters are linked in a source to source way. 
+
+12. Pull in the `replication_manager.sh` script. On each node, perform the following steps:
+
+    ```bash
+    # cd /usr/local/bin
+    # wget https://github.com/y-trudeau/Mysql-tools/raw/master/PXC/replication_manager.sh
+    # chmod u+x replication_manager.sh
+    ```
+
+    When executed for the first time, the replication manager will detect the current replication links and insert rows in to the `percona.replication table`. In order to avoid issues, start with the replica nodes. The mysql credentials should be specified in the `/root/.my.cnf` file. On these nodes (DC1-1, DC2-1 and DC3-1), execute the script manually:
```

Comment:
```suggestion
    When executed for the first time, the replication manager detects the current replication links and inserts rows in to the `percona.replication table`. In order to avoid issues, start with the replica nodes. The mysql credentials should be specified in the `/root/.my.cnf` file. On these nodes (DC1-1, DC2-1 and DC3-1), execute the script manually:
```